.projen/deps.json

```diff
@@ -87,7 +87,7 @@
     },
     {
       "name": "aws-cdk-lib",
-      "version": "^2.88.0",
+      "version": "^2.118.0",
```

Comment:
Is this the lowest version of the CDK required?

Reply:
I found that the solution does not compile below `2.114.0`

Reply:
done in [e8ebbf5](https://github.com/awslabs/generative-ai-cdk-constructs/pull/180/commits/e8ebbf58d89f9699465624722f45c3eb9401228f)